src/notification-hub/azext_notification_hub/tests/latest/test_notificationhubs_scenario.py

```diff
@@ -0,0 +1,179 @@
+                 '--namespace-name {namespace-name} '
+                 '--notification-hub-name {notification-hub-name} '
+                 '--name "my-hub-listen-key"',
+                 checks=[JMESPathCheck('name', 'my-hub-listen-key')])
+
+        self.cmd('az notification-hub authorization-rule list '
+                 '--resource-group {rg} '
+                 '--namespace-name {namespace-name} '
+                 '--notification-hub-name {notification-hub-name}',
+                 checks=[JMESPathCheckExists("[0].rights")])
+
+        self.cmd('az notification-hub authorization-rule list-keys '
+                 '--resource-group {rg} '
+                 '--namespace-name {namespace-name} '
+                 '--notification-hub-name {notification-hub-name} '
+                 '--name "my-hub-listen-key"',
+                 checks=[JMESPathCheckExists('primaryConnectionString')])
+
+        self.cmd('az notification-hub credential gcm update '
+                 '--resource-group {rg} '
+                 '--namespace-name {namespace-name} '
+                 '--notification-hub-name {notification-hub-name} '
+                 '--google-api-key "XXXXX"',
+                 checks=[])
+
+        self.cmd('az notification-hub credential list '
+                 '--resource-group {rg} '
+                 '--namespace-name {namespace-name} '
+                 '--notification-hub-name {notification-hub-name}',
+                 checks=[JMESPathCheckExists('gcmCredential.googleApiKey')])
+
+        # This test needs to use an Android App to receive notification:
+        # https://docs.microsoft.com/en-us/azure/notification-hubs/notification-hubs-android-push-notification-google-fcm-get-started
+        # self.cmd('az notification-hub test-send '
+        #     '--resource-group {rg} '
+        #     '--namespace-name {namespace-name} '
+        #     '--notification-hub-name {notification-hub-name} '
+        #     '--notification-format gcm '
+        #     r'--payload "{\"data\":{\"message\":\"test notification\"}}"',
+        #     checks=[])
+
+        self.cmd('az notification-hub show '
+                 '--resource-group {rg} '
+                 '--namespace-name {namespace-name} '
+                 '--name {notification-hub-name}',
+                 checks=[JMESPathCheck('name', self.kwargs.get('notification-hub-name', ''))])
+
+        self.cmd('az notification-hub list '
+                 '--resource-group {rg} '
+                 '--namespace-name {namespace-name}',
+                 checks=[])
+
+        self.cmd('az notification-hub namespace show '
+                 '--resource-group {rg} '
+                 '--name {namespace-name}',
+                 checks=[JMESPathCheck('name', self.kwargs.get('namespace-name', ''))])
+
+        self.cmd('az notification-hub namespace list '
+                 '--resource-group {rg}',
+                 checks=[])
+
+        # self.cmd('az notification-hub authorization-rule regenerate-keys '
```

Comment:
Is there any reason to comment these two tests?

Reply:
These two are not necessary for a top usage scenario.

Reply:
let's merge this PR